src/Og.php

```diff
@@ -205,6 +212,54 @@ public static function isGroupAudienceField(FieldDefinitionInterface $field_conf
   }
 
   /**
+   * Get the name of the group-audience type field.
+   *
+   * @param $entity_type
+   *   The entity type.
+   * @param $bundle_name
+   *   The bundle name to be checked.
+   * @param $group_type
+   *   Filter list to only include fields referencing a specific group type.
+   * @param $group_bundle
+   *   Filter list to only include fields referencing a specific group bundle.
+   *   Fields that do not specify any bundle restrictions at all are also
+   *   included.
+   *
+   * @return
+   *   Array keyed with the field name and the field label as the value.
+   */
+  public static function getGroupAudienceFields($entity_type, $bundle_name, $group_type = NULL, $group_bundle = NULL) {
```

Comment:
This method name is a bit confusing. Since this returns an array and Fields is plural this seems to indicate that it returns multiple (?) group audience fields. Maybe better would be `getGroupAudienceFieldName()`.

Initially I thought it would be more useful to return the full field definition, but I suppose the machine name and label are the only interesting properties on it anyway.